client/my-sites/customer-home/components/full-screen-launchpad.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import LaunchpadPreLaunch from '../cards/launchpad/pre-launch';
+
+export const FullScreenLaunchpad = ( { onClose }: { onClose: () => void } ) => {
+	const { __ } = useI18n();
+
+	return (
+		<div css={ { display: 'flex', flexDirection: 'column', alignItems: 'center', gap: '24px' } }>
```

Comment:
It's cool to see an inline approach to styling here. Is this for the upcoming CSS overwrites with the new Launchpad design?

This also gets me thinking what is the best way to transition for the upcoming style changes 🤔